src/constants.ts

```diff
@@ -41,31 +41,8 @@ export const ARWEAVE_TAG_APP_VERSION = 2;
 export const PROTOCOL_DEFAULT_CHAIN_ID_INDICES = [1, 10, 137, 288, 42161];
 
 // See src/utils/NetworkUtils for helpers.
-export const PRODUCTION_CHAIN_IDS = [
-  acxConstants.MAINNET_CHAIN_IDs.MAINNET,
-  acxConstants.MAINNET_CHAIN_IDs.OPTIMISM,
-  acxConstants.MAINNET_CHAIN_IDs.POLYGON,
-  acxConstants.MAINNET_CHAIN_IDs.ZK_SYNC,
-  acxConstants.MAINNET_CHAIN_IDs.BASE,
-  acxConstants.MAINNET_CHAIN_IDs.ARBITRUM,
-  acxConstants.MAINNET_CHAIN_IDs.LINEA,
-];
-
-export const TESTNET_CHAIN_IDS = [
-  acxConstants.TESTNET_CHAIN_IDs.GOERLI,
-  acxConstants.TESTNET_CHAIN_IDs.ZK_SYNC_GOERLI,
-  acxConstants.TESTNET_CHAIN_IDs.ZK_SYNC_SEPOLIA,
-  acxConstants.TESTNET_CHAIN_IDs.OPTIMISM_GOERLI,
-  acxConstants.TESTNET_CHAIN_IDs.MUMBAI,
-  acxConstants.TESTNET_CHAIN_IDs.POLYGON_AMOY,
-  acxConstants.TESTNET_CHAIN_IDs.BASE_GOERLI,
-  acxConstants.TESTNET_CHAIN_IDs.BASE_SEPOLIA,
-  acxConstants.TESTNET_CHAIN_IDs.ARBITRUM_GOERLI,
-  acxConstants.TESTNET_CHAIN_IDs.ARBITRUM_SEPOLIA,
-  acxConstants.TESTNET_CHAIN_IDs.SEPOLIA,
-  acxConstants.TESTNET_CHAIN_IDs.OPTIMISM_SEPOLIA,
-  acxConstants.TESTNET_CHAIN_IDs.LINEA_GOERLI,
-];
+export const PRODUCTION_CHAIN_IDS = Object.values(acxConstants.MAINNET_CHAIN_IDs);
+export const TESTNET_CHAIN_IDS = Object.values(acxConstants.TESTNET_CHAIN_IDs);
```

Comment:
Hm, I see above that we already re-export `MAINNET_CHAIN_IDS` and `TESTNET_CHAIN_IDS` directly from constants-v3. I'll instead remove these lines because it's ++confusing otherwise.

Reply:
https://github.com/across-protocol/sdk-v3/pull/654/commits/e9eae24c9da779f9e3ded2a8a9d2a530b72dd239